tutorials/4.0_results_analysis.ipynb

```diff
@@ -0,0 +1,138 @@
+      "\u001b[1;32m/home/luke/code/cyclops/tutorials/4.0 results analysis.ipynb Cell 2\u001b[0m line \u001b[0;36m1\n\u001b[1;32m     <a href='vscode-notebook-cell:/home/luke/code/cyclops/tutorials/4.0%20results%20analysis.ipynb#W2sZmlsZQ%3D%3D?line=13'>14</a>\u001b[0m true_temp_field \u001b[39m=\u001b[39m pickle_manager\u001b[39m.\u001b[39mread_file(\u001b[39m'\u001b[39m\u001b[39mresults/temp_line_field.pickle\u001b[39m\u001b[39m'\u001b[39m)\n\u001b[1;32m     <a href='vscode-notebook-cell:/home/luke/code/cyclops/tutorials/4.0%20results%20analysis.ipynb#W2sZmlsZQ%3D%3D?line=14'>15</a>\u001b[0m grid \u001b[39m=\u001b[39m pickle_manager\u001b[39m.\u001b[39mread_file(\u001b[39m'\u001b[39m\u001b[39mresults/temp_line_points.pickle\u001b[39m\u001b[39m'\u001b[39m)\n\u001b[0;32m---> <a href='vscode-notebook-cell:/home/luke/code/cyclops/tutorials/4.0%20results%20analysis.ipynb#W2sZmlsZQ%3D%3D?line=15'>16</a>\u001b[0m res \u001b[39m=\u001b[39m pickle_manager\u001b[39m.\u001b[39;49mread_file(\u001b[39m'\u001b[39;49m\u001b[39mresults/temp_1D_res_RBF5.pickle\u001b[39;49m\u001b[39m'\u001b[39;49m)\n\u001b[1;32m     <a href='vscode-notebook-cell:/home/luke/code/cyclops/tutorials/4.0%20results%20analysis.ipynb#W2sZmlsZQ%3D%3D?line=16'>17</a>\u001b[0m field_bounds \u001b[39m=\u001b[39m true_temp_field\u001b[39m.\u001b[39mget_bounds()\n\u001b[1;32m     <a href='vscode-notebook-cell:/home/luke/code/cyclops/tutorials/4.0%20results%20analysis.ipynb#W2sZmlsZQ%3D%3D?line=17'>18</a>\u001b[0m sensor_bounds \u001b[39m=\u001b[39m field_bounds\u001b[39m+\u001b[39mnp\u001b[39m.\u001b[39marray([[\u001b[39m1\u001b[39m], [\u001b[39m-\u001b[39m\u001b[39m1\u001b[39m]])\u001b[39m*\u001b[39m\u001b[39m0.002\u001b[39m\n",
+      "File \u001b[0;32m~/code/cyclops/src/cyclops/object_reader.py:39\u001b[0m, in \u001b[0;36mPickleManager.read_file\u001b[0;34m(self, file_path)\u001b[0m\n\u001b[1;32m     30\u001b[0m \u001b[39m\u001b[39m\u001b[39m\"\"\"Read an object instance from a .pickle file and return it.\u001b[39;00m\n\u001b[1;32m     31\u001b[0m \n\u001b[1;32m     32\u001b[0m \u001b[39mArgs:\u001b[39;00m\n\u001b[0;32m   (...)\u001b[0m\n\u001b[1;32m     36\u001b[0m \u001b[39m    object_to_return (object): the object instance loaded from file.\u001b[39;00m\n\u001b[1;32m     37\u001b[0m \u001b[39m\"\"\"\u001b[39;00m\n\u001b[1;32m     38\u001b[0m file_path \u001b[39m=\u001b[39m pathlib\u001b[39m.\u001b[39mPath(file_path)\n\u001b[0;32m---> 39\u001b[0m \u001b[39mwith\u001b[39;00m \u001b[39mopen\u001b[39;49m(file_path, \u001b[39m\"\u001b[39;49m\u001b[39mrb\u001b[39;49m\u001b[39m\"\u001b[39;49m) \u001b[39mas\u001b[39;00m file:\n\u001b[1;32m     40\u001b[0m     object_to_return \u001b[39m=\u001b[39m pickle\u001b[39m.\u001b[39mload(file)\n\u001b[1;32m     41\u001b[0m \u001b[39mreturn\u001b[39;00m object_to_return\n",
+      "\u001b[0;31mFileNotFoundError\u001b[0m: [Errno 2] No such file or directory: 'results/temp_1D_res_RBF5.pickle'"
```

Comment:
Unsure, in other cases the tutorial files are generated in earlier tutorials. Think it would be better to make the tutorials self-contained honestly. In this specific case it seems whatever this file was it's no longer being generated. Requires further investigation. I'll record this as an issue.